src/js/directives/dir-assetDeletion.js

```diff
@@ -0,0 +1,9 @@
+const app = angular.module('materia')
+app.directive('deleteAction', () => ({
+	restrict: 'A',
+	link(scope, element, attrs) {
+		// const onDeleteHandler = scope.$eval(attrs.deleteAction)
+		// element.bind('change', onDeleteHandler)
+		// element.bind('drop', onDeleteHandler)
+	}
+}))
```

Comment:
Is this directive even being used? It doesn't look like it would do anything even if it was.

If this is not necessary, delete it.

Reply:
Deleted the file.